Configuration.cpp

```diff
@@ -52,13 +52,40 @@ bool Configuration::readConfigFile(const char *path) {
   } else
     nfq_queue_id = root["queue_id"].asUInt();
 
+  if(root["markers"].empty()) {
```

Comment:
Can you please make markers optional? This looks like a quick fix. This to avoid existing users to modify their configuration file

Reply:
Ok, i have fixed it